client/lib/plugins/actions.js

```diff
@@ -211,6 +211,10 @@ PluginsActions = {
 			return queueSitePluginActionAsPromise( wpcom.pluginsInstall.bind( wpcom ), site.ID, plugin.slug );
 		};
 
+		update = function( pluginData ) {
+			return queueSitePluginActionAsPromise( wpcom.pluginsUpdate.bind( wpcom ), site.ID, pluginData.id );
```

Comment:
No, it just returns the plugin info (with a `log` of "No update needed"). It's not really an error state, since the plugin is already in the state we want. The process continues on to activating etc.